### PR TITLE
[#37894] Amend clipping plane direction in bcf viewer

### PR DESCRIPTION
**Pull request by Kharonus**
https://community.openproject.org/work_packages/37894

Added configurational flag to bcf viewer
- on loading: clipping direction is reversed
- on saving: clipping direction is reversed
- fixed maxlen of jshint to maxlen of editorconfig
- removed duplicate key in eslintrc

### CODE REVIEW

File: frontend/.eslintrc.js [hunk 1]
```diff
@@ -101,7 +101,6 @@ module.exports = {
         "no-empty": "error",
         "no-eval": "error",
         "no-new-wrappers": "error",
-        "no-redeclare": "error",
```
Comment (Kharonus) [inline frontend/.eslintrc.js]: this key was duplicate. it is redeclared in line 136

File: frontend/.jshintrc [hunk 2]
```diff
@@ -46,7 +46,7 @@
   "noarg": true,
   "noempty": true,
   "nonew": true,
-  "maxlen": 100,
+  "maxlen": 120,
```
Comment (Kharonus) [inline frontend/.jshintrc:49]: this value was in conflict with the maxlen in editorconfig

**Comment by Kharonus** [comment]
This changes breaks all legacy endpoints created from the bcf viewer (clipping direction is reversed on load). The problem (and the implications) and the related solution are described in the ticket: https://community.openproject.org/projects/bim-viewer/work_packages/37894/activity

If we agree to have this as the most pragmatic solution, I will "dedraft" this PR.

**Review by wielinde — APPROVED** [review]
Looks good to me. The changes in .eslintrc.js might cause conflicts later in case linting gets changed upstream and this PR hangs around for longer before merging.

However, let's wait with merging until we have our migration strategy for existing viewpoints clear.

**Review by wielinde — APPROVED** [review]
LGTM

**Pull request merged**